contracts/WhitePaperInterestRateModel.sol

```diff
@@ -56,37 +58,42 @@ contract WhitePaperInterestRateModel is InterestRateModel {
      * @param borrows The amount of borrows in the market
      * @param reserves The amount of reserves in the market
      * @param reserveFactorMantissa The current reserve factor for the market
+     * @param badDebt The amount of badDebt in the market
      * @return The supply rate percentage per block as a mantissa (scaled by EXP_SCALE)
      */
     function getSupplyRate(
         uint256 cash,
         uint256 borrows,
         uint256 reserves,
-        uint256 reserveFactorMantissa
+        uint256 reserveFactorMantissa,
+        uint256 badDebt
     ) public view override returns (uint256) {
         uint256 oneMinusReserveFactor = MANTISSA_ONE - reserveFactorMantissa;
-        uint256 borrowRate = getBorrowRate(cash, borrows, reserves);
+        uint256 borrowRate = getBorrowRate(cash, borrows, reserves, badDebt);
         uint256 rateToPool = (borrowRate * oneMinusReserveFactor) / EXP_SCALE;
-        return (utilizationRate(cash, borrows, reserves) * rateToPool) / EXP_SCALE;
+        uint256 rate = ((borrows * EXP_SCALE) / (cash + borrows + badDebt - reserves));
```

Comment:
Same here, I think we can avoid the `/ EXP_SCALE` and the `* EXP_SCALE`

Reply:
I guess, we can't remove  `* EXP_SCALE` from `rate` as `(borrows / (cash + borrows + badDebt - reserves))` will be in fraction.